doc/README.md

```diff
@@ -9,6 +9,10 @@ Therefore you need to have the following installed:
 - [Docker Compose](https://docs.docker.com/compose/install/)
 - [Drone](https://docs.drone.io/cli/install/)
 
+If you are going to use Django tranlations, you must install the GNU **gettext** utilities
```

Comment:
typo **translations**

The next paragraph in the README still refers to Docker. I would move this gettext paragraph a bit further so that it does not break the flow of the Docker paragraphs.